activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -446,12 +446,15 @@ def build_insert_sql(insert) # :nodoc:
         sql
       end
 
-      private
-        def check_version
-          if postgresql_version < 90300
-            raise "Your version of PostgreSQL (#{postgresql_version}) is too old. Active Record supports PostgreSQL >= 9.3."
-          end
+      def check_version!
+        if postgresql_version < 90300
+          raise "Your version of PostgreSQL (#{postgresql_version}) is too old. Active Record supports PostgreSQL >= 9.3."
```

Comment:
Why not 9.5: https://blog.2ndquadrant.com/what-is-select-skip-locked-for-in-postgresql-9-5/

Reply:
This PR is just moving the check above private, we can change the message in a later PR.

Reply:
How is that related?